### ISSUE TITLE
Open links in default browser

### ISSUE THREAD

**Issue by KazeJiyu** [ShuttleLtd/Shuttle#12]
At this time, clicking on a link opens it in Shuttle's current view. However, it could be helpful to provide a way to open the link with user's default browser.

This could be achieved by, for exemple: 
- Add a "Open in browser" item in the context menu associated with links,
- Add a shortkey, such as Ctrl, that would change the way a link is opened when pressed.

@Vahelnir, do you think it would be a bit harsh to implement or rather trivial ?

**Comment by Vahelnir** [comment]
It is rather simple, we just have to add an eventlistener to the `click` event on `<a href="#"></a>` and just [open the url in the default browser](https://github.com/electron/electron/blob/master/docs/api/shell.md#shellopenexternalurl-options-callback)

**Comment by KazeJiyu** [comment]
And what about enriching the context menu ? 

We could [build it from scratch](https://www.sitepoint.com/building-custom-right-click-context-menu-javascript/), but it would be laborious, and someone has already [built an an extension for Electron](https://github.com/sindresorhus/electron-context-menu). Have you already used it ?

**Comment by Vahelnir** [comment]
Nope, never used it before. We should try to implement something cool with it first, and then see if it is what we need.
